views/includes/metadata.hbs

```diff
@@ -0,0 +1,56 @@
+<!-- Schema.org markup for Google+ -->
+<meta itemprop="name" content="{{title}}">
+<meta itemprop="description" content="{{description}}">
+<meta itemprop="image" content="{{leader}}">
+
+<!-- Open Graph data -->
+<meta property="fb:app_id" content="205743429831145" />
+<meta property="og:title" content="{{title}}" />
+<meta property="og:type" content="article" />
+<meta property="og:image" content="{{leader}}" />
+<meta property="og:url" content="{{url}}" />
+<meta property="og:description" content="{{description}}" />
+<meta property="og:site_name" content="PWA Directory" />
+<meta property="article:published_time" content="{{datePublished}}" />
+<meta property="article:modified_time" content="{{dateModified}}" />
+
+<!-- Metadata required by Google -->
+<script type="application/ld+json">
+{
+  "@context": "http://schema.org",
+  "@type": "NewsArticle",
```

Comment:
I will change it to CollectionPage, we can decide later what is better if we want to have rich results.